docs/pages/application-access/guides/api-access.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Using Application Access to Access APIs
+description: How to access RESTful APIs using Teleport Application Access
+---
+
+# API Access
+
+Application Access can be used to access applications' RESTful APIs with
+tools like [curl](https://man7.org/linux/man-pages/man1/curl.1.html) or
+[Postman](https://www.postman.com/).
+
+## Prerequisites
+
+We will assume that you followed the [Getting Started](../getting-started.mdx)
+guide or the general [App Access Usage](./usage.mdx) guide to connect the web
+application providing an API to Teleport.
+
+For the simplicity of the guide, we will be using Grafana running in a
+[Docker container](https://grafana.com/docs/grafana/latest/installation/docker/)
+and run API queries against it. You can launch Grafana too with a single Docker
+command:
+
+```sh
+$ docker run -d -p 3000:3000 grafana/grafana
+```
+
+Connect Grafana to your Teleport cluster by adding the following section in
+the Teleport App Service configuration file:
+
+```yaml
+app_service:
+  enabled: yes
+  apps:
+  - name: "grafana"
+    description: "Test Grafana server"
+    uri: "http://localhost:3000"
+```
+
+## Accessing API
+
+Log into your Teleport cluster and view available applications:
+
+```sh
+$ tsh login --proxy=teleport.example.com:3080
+$ tsh app ls
+Application Description         Public Address               Labels
+----------- ------------------- ---------------------------- ------
+grafana     Test Grafana server grafana.teleport.example.com
+```
+
+Retrieve short-lived X.509 certificate for the application:
+
+```sh
+$ tsh app login grafana
+Logged into app grafana. Example curl command:
+
+curl \
+  --cacert /Users/alice/.tsh/keys/teleport.example.com/certs.pem \
+  --cert /Users/alice/.tsh/keys/teleport.example.com/alice-app/cluster-name/grafana-x509.pem \
+  --key /Users/alice/.tsh/keys/teleport.example.com/alice \
+  https://grafana.teleport.example.com:3080
+```
+
+The login message shows an example `curl` command you can run to call the
```

Comment:
Could use note admonition instead of using note word if you wanted to highlight the topic?